mobile-widgets/qml/DivePlannerSetup.qml

```diff
@@ -265,7 +265,7 @@ Kirigami.ScrollablePage {
 					text: qsTr("SAC factor")
 				}
 				TemplateSpinBox {
-					from: 20
+					from: 1
```

Comment:
when would it make sense for this factor to be `<1`?

Reply:
for reference the value is presented with 1 decimal, so the range on the screen is 0.1 to 9.90.

The desktop allows a range of 2.0 to 10.0.

I have never heard of a negative SAC factor, it would imply you do not consume air, but produce air. Maybe it is something that e.g. @atdotde can tell us.

Reply:
still curious about this one because I don't understand the reason for this change... I could see allowing factors `>= 1.0`, but I think I don't understand `>= 0.1`